src/networking/payload/bits.rs

```diff
@@ -0,0 +1,28 @@
+
+#[derive(PartialEq, Debug)]
+pub struct Bits{
+  bit0: bool,
+  bit1: bool,
+  bit2: bool,
+  bit3: bool,
+  bit4: bool,
+  bit5: bool,
+  bit6: bool,
+  bit7: bool,
+}
+
+
+impl Bits{
+  pub fn from(num:u8) -> Self{
```

Comment:
What does this do?

Reply:
the bits struct is a construct used in py-ipv8 which i kept in mostly to keep their features similar. it's a struct containing booleans representing each bit in an u8. the from(u8) just creates such a struct from an u8

Reply:
I would suggest adding a comment denoting this as this isn't self-evident

Reply:
done that